### ISSUE TITLE
Users on upload white list should still be able to access subject upload interface

### ISSUE THREAD

**Issue by srallen** [zooniverse/Panoptes-Front-End#3731]
## Expected behavior
Users on whitelist should be able to access subject upload interface.

## Current behavior
_Please include any error messages from the browser console and/or screenshots_
See original opened issue at zooniverse/Panoptes#2301
Regression from #3723

**Comment by mschwamb** [comment]
@srallen It's still saying I can't upload subjects. I thought by now the servers would have updated.  Should I check again tomorrow?

**Comment by srallen** [comment]
@mschwamb the pull request, #3732, hasn't been reviewed and merged yet. The link I asked you to take a look at is on staging using a query parameter, `?env=production`, so that it uses the production database temporarily for testing purposes. 

I can't merge my own pull request. It's a github option that we enabled to make sure we have at least one other developer look over pull requests. It likely won't merge till tomorrow sometime since I requested review from @simoneduca since he last worked on that part of the code base and today is a UK bank holiday.
